tokens.xml

```diff
@@ -1352,6 +1370,32 @@ This creature can block only creatures with flying.</text>
             <token>1</token>
             <tablerow>2</tablerow>
         </card>
+        <card>
+            <name>Citizen Token  </name>
+            <text></text>
+            <prop>
+                <colors>GW</colors>
+                <type>Token Creature — Citizen</type>
+                <maintype>Creature</maintype>
+                <cmc>0</cmc>
+                <pt>1/1</pt>
+            </prop>
+            <set picURL="https://media.wizards.com/2022/snc/en_SfS0UTN94g.png">SNC</set>
+            <reverse-related count="2">Cabaretti Charm</reverse-related>
+            <reverse-related attach="attach">Citizen's Crowbar</reverse-related>
+            <reverse-related>Courier's Briefcase</reverse-related>
+            <reverse-related>Darling of the Masses</reverse-related>
+            <reverse-related>Exhibition Magician</reverse-related>
+            <reverse-related>Halo Fountain</reverse-related>
+            <reverse-related>Luxurious Libation</reverse-related>
+            <reverse-related count="x">Rabble Rousing</reverse-related>
+            <reverse-related>Rakish Revelers</reverse-related>
+            <reverse-related>Sanctuary Warden</reverse-related>
+            <reverse-related count="x" exclude="exclude">Stimulus Package</reverse-related>
```

Comment:
I don't think this is an x ability, just exclude imo

Reply:
Yes, I was annoyed by this when I tested it.

Reply:
```suggestion
```

I do agree, only exclude. 👍